### PR TITLE
Updating documentation to support percentiles chart.

### PR DESCRIPTION
**Pull request by merryt**
https://www.notion.so/memfault/2023-08-03-Metric-Percentiles-Chart-Roll-out-8f4a157f4f18499fa3a5a5e63d516fe8?pvs=4#f073033e688e4797a00f72d2e6148808

### CODE REVIEW

**Comment by merryt** [comment]
[Additional reading from notion](https://www.notion.so/memfault/2023-08-03-Metric-Percentiles-Chart-Roll-out-8f4a157f4f18499fa3a5a5e63d516fe8?pvs=4#5d54c729416e40898f3dc8e3a1f0dc07),

**Comment by utkuoymak** [comment]
Good addition for the population charts however since this is the first ever "percentile" chart we have, it might be good to add some images or a gif to show what it looks like. Using the `battery percentige` metric in smart sink project has the "best" looking data we have for percentiles right now. 
image

**Comment by merryt** [comment]
@utkuoymak Sounds good. Once we get the `x samples over 7 days` content into the aside I will snap a screenshot and put it in.

File: docs/platform/charts.mdx [hunk 1]
```diff
@@ -77,6 +77,7 @@ upload data for a given metric before a chart for that metric can be created.
 | Mean         | Arithmetic mean of all readings.               |
 | Max          | The highest reading received.                  |
 | Min/Mean/Max | All of the above combined into a single chart. |
+| Percentiles  | Distribution of precentiles of type of reading |
```
Comment (ejohnso49) [inline docs/platform/charts.mdx]:
```suggestion
| Percentiles  | Distribution of percentiles of type of reading |
```

Comment (elisekain) [reply docs/platform/charts.mdx]: Thank you Eric! Caught this in a subsequent commit

Comment (ejohnso49) [reply docs/platform/charts.mdx]: Oh perfect, sorry about that!

**Review by utkuoymak — APPROVED** [review]

**Pull request merged**